### ISSUE TITLE
Signal handling

### ISSUE THREAD

**Issue by kmaehashi** [sensorbee/sensorbee#153]
Currently SensorBee does not handle signals.
I think it's better to trap Ctrl-C and shutdown gracefully (i.e., call Terminate for every user-defined elements in topologies so that it can perform cleanup).

**Comment by nobu-k** [comment]
Yes, we need to handle signals properly. I was planning to do it after SensorBee drops support of old Go versions because Go's standard HTTP server now supports graceful shutdown. So, we should be able to work on this.
